### PR TITLE
Add rosa2019.05, fix repos of rosa2019.0

### PR DESCRIPTION
**Pull request by mikhailnov**

### CODE REVIEW

File: repos.d/rpm/rosa.yaml [hunk 1]
```diff
@@ -60,7 +63,8 @@
   color: '3757a1'
   minpackages: 6500
   sources:
-    - name: [ main/release, main/testing, main/updates, non-free/release, non-free/testing, non-free/updates, restricted/release, restricted/testing, restricted/updates, contrib/release, contrib/testing, contrib/updates ]
+    # see comments bellow in rosa2019.05
```
Comment (AMDmi3) [inline repos.d/rpm/rosa.yaml:66]: Please duplicate the comment here, as repository entries are managed independently

File: repos.d/rpm/rosa.yaml [hunk 2]
```diff
@@ -74,7 +78,34 @@
...
+  statsgroup: Rosa
+  family: rosa
+  ruleset: [rosa, rpm]
+  color: '3757a1'
+  minpackages: 6500
+  sources:
+    # - rosa2019.0 and rosa2019.05 are platforms for special purposes (certification), restricted is merged into
+    # main and is removed because it makes no sense there as US patent do not work in Russia which is the only
+    # region where certified distros will be distributed;
+    # - contrib does not exist there;
```
Comment (AMDmi3) [inline repos.d/rpm/rosa.yaml:94]: `contrib/release` is non-empty for both 2019.0 and 2019.05, please elaborate

File: repos.d/rpm/rosa.yaml (hunk 1 above)
Comment (mikhailnov) [reply repos.d/rpm/rosa.yaml]: Done

File: repos.d/rpm/rosa.yaml (hunk 2 above)
Comment (mikhailnov) [reply repos.d/rpm/rosa.yaml:98]: thanks for noting this, they have been cleaned from some random packages

**Comment by mikhailnov** [comment]
Is any action needed from my side?

**Comment by AMDmi3** [comment]
404 http://abf-downloads.rosalinux.ru/rosa2019.05/repository/SRPMS/main/testing/repodata/repomd.xml

**Comment by mikhailnov** [comment]
Unfortunately it has been decided to clode public access to repos of those platfroms, now all URLs to them will give error 403 (git is still public, GPL is not violated). Closing this PR as this does not make any sense now. rosa2019.0 needs to be removed from Repology.

**Pull request closed without merging**